### ISSUE TITLE
Slides not ordered by cmdline

### ISSUE THREAD

**Issue by Phelimb** [Blahah/slidewinder#6]
```
 slidewinder   --collection ./slides/   --slides title,code,data,list   --title title   --author phelimb   --output test_deck
```

Puts the slides in the order code, data, list, title. There doesn't seem to be a way of controlling the ordering of the slides? 

Looks like they're maybe being ordered by creation date?

**Comment by blahah** [comment]
Thanks for reporting! Fixed in aeab3ad23c2fe882dd253011eb56ccc78dd0f10d, and I pushed a new version (v0.0.3) containing the fix.

**Comment by blahah** [comment]
This issue was moved to slidewinder/slidewinder#6
